### DISCUSSION TITLE
Need help Android car

### DISCUSSION THREAD

**Discussion by MgsmSC** [community/community#64796] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need help changing the language of the Android car. Help with finding a specialist

**Comment by agetbase** [comment]
If you are meaning to `Android Auto`. The language of Android Auto typically follows the language setting of the connected smartphone. Therefore, to change the language of Android Auto, you would need to change the language setting on your smartphone.

Here's how to change the language setting on an Android smartphone:

1. **Go to Phone Settings**: Open the 'Settings' app on your smartphone.
2. **Navigate to System Settings**: Find and tap on an option like 'System' or 'General Management'. (The name might vary depending on the device and version.)
3. **Choose Language & Input**: Tap on an option like 'Languages & Input' or 'Language & Keyboard'.
4. **Select Language**: Tap on 'Language' or 'Add a Language'.
5. **Pick the Desired Language**: Find and select the language you want from the list.
6. **Save Changes**: Save the changes if necessary.
Now that the language of the connected smartphone has been changed, Android Auto should also display in that language.

> Please note, not all languages might be supported by Android Auto. You can check the list of supported languages on the official Android Auto website or its documentation.

**Comment by Akash1134** [comment]
Hi @MgsmSC ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
